XO.py

```diff
@@ -0,0 +1,157 @@
+from random import shuffle
+
+# size of board
+N = 3
+
+# Utility functions
+
+
+def print_board(board):
+    print()
+    print('Board:')
+    for line_ in board:
+        for item in line_:
+            print(item, end='')
+        print()
+    print()
+
+
+def check_input_validity(input_type, curr_player):
+    not_valid = True
+    while not_valid:
+        print('{}, please insert the {} number'.format(curr_player, input_type))

```

Comment:
Try and use `f-strings`, they look much nicer!
(new in Python 3.6)
Example:
`print(f'{curr_player}, please insert the {input_type} number')`

Reply:
Done